cves/kernel/CVE-2021-3573.yml

```diff
@@ -184,9 +192,11 @@ specification:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  note:
-  answer:
-subsystem:
+  note: |
+    it is noted that the fix changes the lock to the correct one for serialization
```

Comment:
I find this statement very vague, it uses general language and could be reworked to be more descriptive.

Reply:
Okay I'll see what I can do